src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Color defaultFigureColor = Color.WHITE;
+    private final int defaultFigureRadius = 10;
+    private final int numberOfFigures = 5;
+    private final int maxSizeParameters = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle(defaultFigureColor, defaultFigureRadius);
+    }
+
+    public Figure getRandomFigure() {
+        final Color randomColor = Color.valueOf(colorSupplier.getRandomColor());

```

Comment:
you wrap String in Color enum and then immediately unwrap it again - makes no sense
```suggestion
        final String color = colorSupplier.getRandomColor();
```